tools/setup_helpers/cuda.py

```diff
@@ -49,11 +68,15 @@ def find_cuda_version(cuda_home):
     CUDA_HOME = None
     CUDA_VERSION = None
 else:
-    CUDA_HOME = os.getenv('CUDA_HOME', '/usr/local/cuda')
+    if IS_WINDOWS:
+        CUDA_HOME = os.getenv('CUDA_PATH', None).replace('\\', '/')
```

Comment:
Why are you using a different env variable for Windows?

Reply:
The variable name is different for Windows. No `CUDA_HOME` for windows but 'CUDA_PATH'.

Reply:
My questions was why can't you use the same variable as on all other systems (to be consistent)? Sorry if my question was unclear